components/Calendar/Calendar.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import Calendar from "react-calendar";
+import "react-calendar/dist/Calendar.css";
+import styled from "styled-components";
+
+export function CalendarFromReact({ appointmentList }) {
+  const [date, setDate] = useState(new Date());
+
+  const filterAppointmentsByDate = (appointmentList, clickedDate) => {
+    return appointmentList.filter((appointment) => {
+      const appointmentDate = new Date(appointment.date);
+      return appointmentDate.toDateString() === clickedDate.toDateString();
+    });
+  };
+
+  const handleClickDay = (clickedDate) => {
+    setDate(clickedDate);
+  };
+
+  const isAppointmentOnDate = (date) => {
+    return appointmentList.some((appointment) => {
+      const appointmentDate = new Date(appointment.date);
+      return appointmentDate.toDateString() === date.toDateString();
+    });
+  };
+
+  const tileContent = ({ date }) => {
+    return isAppointmentOnDate(date) ? (
+      // <button style={{ backgroundColor: "green" }} />
```

Comment:
ich würde eigentlich sagen, lösch das bitte, aber du schreibst in der Beschreibung des PR (wenn ich das richtig verstanden habe), dass du damit weiterarbeiten möchtest, also ist das wohl ok ;) allerdings bitte keine Inline-Styles verwenden - du kannst das über styled components lösen :)

Reply:
Aber ich bekomme hier ein return, was nur die Farbe ändern soll oder etwas hinzufügt. muss ich dir mal zeigen, fürs bessere verständis.  : )

Reply:
wenns soweit ist gerne :)